### PR TITLE
"Lava Puddles" by smugler5

### PR DESCRIPTION
**Pull request by Avyukt1**
screenshot
Done with debugging the files. Now it is ready for testing.

### CODE REVIEW

File: lava_puddles/map.conf [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+team.2 = blue
+rotation = z
+name = Lava Puddles
+author = smugler5
+h = 140
+team.1.color = red
+team.2.color = blue
+team.2.pos = (101,1,-102)
+team.1 = red
+licence = CC BY-SA
```
Comment (JostP) [inline lava_puddles/map.conf:13]: license version?

Comment (Avyukt1) [reply lava_puddles/map.conf]: Edited.

**Comment by Avyukt1** [comment]
So, did the testers test the map?

**Comment by Avyukt1** [comment]
This map has lots of lava,
screenshot_20201109_191659
This map also has a castle-like construction.
screenshot_20201109_191931
Diamond is hidden in a small hole that goes ender the ground.

**Comment by JostP** [comment]
How much diamond and mese?

**Comment by Avyukt1** [comment]
There are only 12 diamond ores in the map.
Totally, the sum of diamond swords craft able is '6' only.

**Comment by Avyukt1** [comment]
> How much diamond and mese?

My map doesn't have mese.
This map only has 12 diamond crystals. Only 6 diamond swords can be crafted.

**Comment by JostP** [comment]
I think you should add more. Also add mese. If there is only 6 diamond swords in-game that means only good players will get them and they will be impossible to be killed.

**Comment by Avyukt1** [comment]
Yes! You are right. But Ii had got a review from Elvis26 and winniepee0 that lots of mese and diamond is bad idea.

- Plains of the Past
- Plain Battle
- River Valley

These maps have the same quantity of mese and diamond.
Shall I remove the treasure instead?

**Comment by Avyukt1** [comment]
Anyway,
I was the latest exported data was not working.
I couldn't see anything in the map after editing the map.

**Comment by Avyukt1** [comment]
Anyway, please test the and confirm me.

**Comment by Avyukt1** [comment]
Anyway, please test the map.
And confirm me.

**Comment by Avyukt1** [comment]
Did you all like it?

**Comment by Avyukt1** [comment]
Did you like my map?

File: lava_puddles/map.conf [hunk 2]
```diff
@@ -0,0 +1,14 @@
+team.1.pos = (-98,1,102)
+r = 115
+initial_stuff = default:pick_steel,default:dirt 40
+team.2 = blue
+rotation = z
+name = Lava Puddles by smugler5
```
Comment (Panquesito7) [inline lava_puddles/map.conf:6]: Adding ` by smugler5` is not needed. `name` is the **map name**.  `author` is for **map creator's name**.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by Avyukt1** [comment]
@Panquesito7 did you test my map?

**Comment by Panquesito7** [comment]
> @Panquesito7 did you test my map?

No, I haven't tested it.

**Comment by Avyukt1** [comment]
> > @Panquesito7 did you test my map?
> 
> No, I haven't tested it.

When will you test it?

**Comment by Avyukt1** [comment]
@JostP Please test my map.

**Comment by Avyukt1** [comment]
@JostP 
I think you will not like any map made by me.
I had created this PR before other PRs and you tested other PRs before mine.
I don't think you'll test my map ever.
If I don't get a message before 5 days, I'll delete this Pull Request.

**Comment by winniepee** [comment]
1. People have tested it, but you just need to wait for few months or so. 
2. If you delete this your map don't have a chance to get it.
3. Try making other maps. Make it special for the players so the map can merge it faster.

Hope you can be patient enough! :)

**Comment by Avyukt1** [comment]
> 1. People have tested it, but you just need to wait for few months or so.
okay!

**Comment by Avyukt1** [comment]
Bruh, I am angry on you @JostP 
you are very partial!

**Comment by JostP** [comment]
@Avyukt1 I don't agree with you.
First, I didn't merge any new map created after yours. Yeah, I merged some quick made PR for current maps fixes because the quality of merged maps is important. If you are impatient and can't wait few more days, do whatever you want, I can't help you. You can leave your PR opened, I don't understand why you closed it.

People are creating a lot of new maps recently what causes long waiting for new maps to be reviewed.
I can't dislike your map if as haven't even tested it yet and if I disliked your map, I would first ask other people if they likes it. From what I see, your map looks interesting especially because it has good ores such as mese and diamonds which make fun matches.

**Comment by Avyukt1** [comment]
> @Avyukt1 I don't agree with you.
> First, I didn't merge any new map created after yours. Yeah, I merged some quick made PR for current maps fixes because the quality of merged maps is important. If you are impatient and can't wait few more days, do whatever you want, I can't help you. You can leave your PR opened, I don't understand why you closed it.
> 
> People are creating a lot of new maps recently what causes long waiting for new maps to be reviewed.
> I can't dislike your map if as haven't even tested it yet and if I disliked your map, I would first ask other people if they like it. From what I see, your map looks interesting especially because it has good ores such as mese and diamonds which make fun matches.

@JostP 
This is what I call partiality or laziness!

**Comment by Avyukt1** [comment]
@JostP how much time does it take to test and review a map?

**Comment by JostP** [comment]
more days because most of the maps are not created correctly and have many bugs

**Comment by Avyukt1** [comment]
> more days because most of the maps are not created correctly and have many bugs

Olliy, winniepee and Elvis26 tested it and they didn't find a bug.

**Comment by Avyukt1** [comment]
Reject my map if you don't even want to see it.

**Pull request closed without merging**